app/javascript/scrollable_pane.js

```diff
@@ -1,18 +1,32 @@
-export function handleResizeOnScrollablePane() {
-  const scrollablePane = document.querySelector('.moj-scrollable-pane');
+export function handleResizeOnScrollablePane(scrollablePane) {
+  if (!scrollablePane) return;
+  const isLargeScreen = window.innerWidth >= 1024;
 
-  if (window.innerWidth >= 1024) {
-      scrollablePane.classList.add('hide-horizontal-scroll');
-      // Force repaint on Safari by changing overflow values slightly
```

Comment:
I think it's really worth keeping this comment. Otherwise it's not at all clear why `hide-horizontal-scroll` can't just be controlled by a media query in CSS instead of javascript, why the overflow styles can't just be added to the hide-horizontal-scroll CSS class, and why we need a setTimeout.

To be honest, I still don't really understand why all this JS is needed, but the comment about force repainting on Safari makes me trust that it's not just because we don't understand how CSS works!

Reply:
Added the comment back, now in Line 12